mne/channels/tests/test_montage.py

```diff
@@ -1205,6 +1205,34 @@ def test_set_montage_with_missing_coordinates():
     )
 
 
+def test_get_channel_coordinates():
+    """Test get montage and its coordinates."""
+    N_CHANNELS = 3
+
+    raw = _make_toy_raw(N_CHANNELS)
+    raw.set_channel_types({ch: 'ecog' for ch in raw.ch_names})
+    ch_names = raw.ch_names
+    n_channels = len(ch_names)
+    ch_coords = np.arange(n_channels * 3).reshape(n_channels, 3)
+    montage = make_dig_montage(
+        ch_pos=dict(zip(ch_names, ch_coords,)),
+        nasion=[0, 1, 0], lpa=[1, 0, 0], rpa=[-1, 0, 0],
+        coord_frame='head'
+    )
+    raw.set_montage(montage)
```

Comment:
make it work with testing data as it contains all kinds of dig points you don't want to loose with get_montage

the test should do:
m = inst.get_montage()
inst.set_montage()
and inst should be unchanged

Reply:
What is the best way to compare if two instances are the same? Is what I am doing currently not enough? 

If not, is there a function in mne-python testing for this?